<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualControlDialog</class>
 <widget class="QDialog" name="ManualControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>628</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>534</width>
    <height>628</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelNumPad">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>260</width>
         <height>300</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelInfo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>260</width>
         <height>300</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>530</width>
         <height>270</height>
        </size>
       </property>
       <property name="title">
        <string>Mouse control</string>
       </property>
       <widget class="QWidget" name="layoutWidget2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>191</width>
          <height>201</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QToolButton" name="toolButton8">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>X - Y+</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="toolButton2">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Y+</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="toolButton10">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>X+Y+</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="toolButton6">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>X -</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="toolButton7">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>X+</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="toolButton9">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>X - Y -</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="toolButton3">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Y -</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QToolButton" name="toolButton11">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>X+Y -</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget4">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>60</y>
          <width>126</width>
          <height>52</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QToolButton" name="toolButton12">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>A+</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton13">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>A -</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget1">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>130</y>
          <width>62</width>
          <height>106</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QToolButton" name="toolButton4">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Z+</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton5">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Z -</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>30</y>
          <width>77</width>
          <height>191</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labelVelocity">
           <property name="text">
            <string>Velocity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox"/>
         </item>
         <item>
          <widget class="QSlider" name="verticalSlider">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>398</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
